lib/assessment.js

```diff
@@ -326,7 +326,7 @@ module.exports = {
                     content.grading.score,
                     content.grading.feedback,
                     content.grading.startTime,
-                    content.grading.endTime,
+                    content.grading.endTime
```

Comment:
Any particular reason? I typically use trailing commas so that future diffs don't include spurious lines due to comma changes.

Reply:
Nope, this was me absentmindedly removing a first failed attempt to do this feature. I'd be in favor of turning on a linter for trailing commas this to enforce it everywhere.

Reply:
Sounds great, I'm strongly in favor of requiring trailing commas on multiline objects/arrays.